Gemfile

```diff
@@ -143,7 +143,7 @@ group :test, :development do
 
   # # Used to test with a real browser
   gem "selenium", "~>0.2.11"
-  gem "selenium-webdriver", "~>2.53.0"
+  gem "selenium-webdriver", "~>2.53.4"
```

Comment:
@dej611 Whenever theres a firefox update I'd have to update that specific gem, so I thought to include it explicitly so I can make those changes explicit